analytics/compfeeds_analytics/lending_borrowing/dynamicLiquidator.py

```diff
@@ -0,0 +1,36 @@
+import pandas as pd
+from datetime import datetime
+import time
+import matplotlib.pyplot as plt
+import numpy as np
+import utils
+from matplotlib import style
+import matplotlib.dates as md 
+import scipy 
+
+class dynamicLiquidator:
+    def __init__(self, dynamicLBProtocol): 
+        self.protocol = dynamicLBProtocol 
+        self.reward = 0 
+
+
+    def check_liquidity_opportunities(self, dynamicLBProtocol): 
```

Comment:
i'd recommend snakecase, but i also don't know what the type of dynamicLBProtocol is

Reply:
it looks like it's the class from the other file.  you shouldn't name a variable the same as the classname